src/features/checker/pages/OverviewPage/components/OverviewPageTitle/OverviewPageTitle.tsx

```diff
@@ -0,0 +1,10 @@
+export const OverviewPageTitle = () => (
+  <div className="flex flex-col gap-[12px]">
+    <div className="font-['DM Sans'] text-2xl font-medium leading-loose text-black">
+      Review applications
+    </div>
+    <div className="font-['DM Sans'] text-base font-normal leading-7 text-[#555555]">
```

Comment:
in the future let's use tailwind.config.ts instead of forcing em here, let's us do darkmode + overrides in client apps but for now this isn't important because it's not the point of this PR!